### PR TITLE
Register tags documentation example updated

### PR DESCRIPTION
**Pull request by kayuapi**

### CODE REVIEW

File: docs/source/tutorials/register-filters-tags.md [hunk 1]
```diff
@@ -6,13 +6,15 @@ title: Register Filters/Tags
 
 ```javascript
```
Comment (harttle) [inline docs/source/tutorials/register-filters-tags.md:7]: update this to typescript

File: docs/source/tutorials/register-filters-tags.md [hunk 2]
```diff
@@ -6,13 +6,15 @@ title: Register Filters/Tags
...
+
 engine.registerTag('upper', {
-    parse: function(tagToken, remainTokens) {
+    parse: function(tagToken: TagToken, remainTokens: TopLevelToken[]) {
         this.str = tagToken.args; // name
     },
-    render: async function(scope, hash) {
-        var str = await this.liquid.evalValue(this.str, scope); // 'alice'
-        return str.toUpperCase()  // 'Alice'
+    render: async function(ctx: Context, emitter: Emitter) {
```
Comment (harttle) [inline docs/source/tutorials/register-filters-tags.md:15]: good catch!

**Review by harttle — CHANGES_REQUESTED** [review]

File: docs/source/tutorials/register-filters-tags.md (hunk 1 above)
Comment (kayuapi) [reply docs/source/tutorials/register-filters-tags.md]: done

File: docs/source/tutorials/register-filters-tags.md (hunk 2 above)
Comment (kayuapi) [reply docs/source/tutorials/register-filters-tags.md:15]: thanks!

**Pull request merged**